webapp/src/index.js

```diff
@@ -0,0 +1,10 @@
+import {id as pluginId} from './manifest';
+
+export default class Plugin {
```

Comment:
Looks like there is no frontend required here -- can you just remove the `webapp` directory altogether?

Reply:
I was thinking about that, but there are some creative ways to suggest channels that will need frontend. For example: pop up in the public channels menu like  "  <- try this channel".
So I left it.

Reply:
Definitely like the forward thinking, but I'm inclined to remove it until there's something other than boilerplate. Adding it back in should be relatively straightforward -- the `Makefile` was built to detect the presence of absence of the `webapp` directory automatically.